registration-processor/registration-processor-message-sender-impl/pom.xml

```diff
@@ -12,7 +12,7 @@
 	</parent>
 
 	<artifactId>registration-processor-message-sender-impl</artifactId>
-	 <version>1.1.5.5</version>
+	 <version>1.1.5.5-p1-SNAPSHOT</version>
```

Comment:
Lets not change this version, also we should ensure this is not getting built

Reply:
done